src/main/java/org/opentripplanner/graph_builder/module/StreetLinkerModule.java

```diff
@@ -53,13 +54,16 @@ public List<String> getPrerequisites() {
   @Override
   public void buildGraph(
     Graph graph,
+    TransitModel transitModel,
     HashMap<Class<?>, Object> extra,
     DataImportIssueStore issueStore
   ) {
+    transitModel.index();
```

Comment:
Is this the defined place in the graph build where the indexing happens or is this just in case someone forgot it in the test?

Reply:
This is actually done for the graph building, as this step is the first one, which needs the indices, and where the transit model has been built.